slo-reporter/base/cronjob.yaml

```diff
@@ -56,7 +56,7 @@ spec:
                   valueFrom:
                     configMapKeyRef:
                       name: thoth
-                      key: deployment-name
+                      key: environment
```

Comment:
Why not keep using deployment name?

Reply:
OK, I see it's related to #298. So what about replacing `-` with `_` or split on `-` following the `deployment_name` naming convention? Introducing another similar variable can cause confusion.

Reply:
Two reasons:

1. `THOTH_ENVIRONMENT`  variable is used to create metric for info.

```
self.environment = os.environ["THOTH_ENVIRONMENT"]
```

```
            self.thoth_weekly_sli = Gauge(
                f"thoth_sli_weekly_{self.environment}",
                "Weekly Thoth Service Level Indicators",
                ["sli_type", "metric_name"],
                registry=self.prometheus_registry,
            )
```
Prometheus metrics do not allow metrics name with `-`, therefore `ocp-stage` cannot be used.

This could be solved by adding `self.environment.replace('-', "_")`

2. but there is a second issue, Prometheus adds label to metrics from DH side. Therefore they contain `job="Thoth Metrics (stage)"` where `test`, `stage`, `prod` is used to differentiate same metric name.
And because `THOTH_ENVIRONMENT`  variable is used to retrieve metrics from Prometheus, even applying solution at 1. `ocp-stage` will not lead to any metrics because they do not exist on Prometheus side

Therefore this would require changes in how metrics are retrieved from DH side where we make sure `job="Thoth Metrics (stage)"` contains the same name as deployment name.

To summarize:

- the current solution in this PR is easy and fast, second solution which requires different changes in different places and new releases can be done as well of course, but it will take some time.

We can create some cards to change that. WDYT? @fridex

Reply:
I think having the split method will not introduce tech dept - we already state the environment in `deployment_name` which is used in the whole Thoth application.

Reply:
https://github.com/thoth-station/slo-reporter/pull/70

Reply:
In this PR I will remove environment variable not used anymore